CHANGELOG.md

```diff
@@ -7,6 +7,24 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.0.0] - 2021-07-17
+
+## Added
+
+- Mix task xcribe.doc
+- Allow multiple endpoints config
+
+### Changed
+
+- Xcribe configuration is scoped by application endpoint module
+- Documentation route foward to Xcribe.Web.Plug you must provide the application
+  endpoint.
+
+### Removed
+
+- The env var config was removed. Now you must use `XCRIBE_ENV` to active xcribe
```

Comment:
So why was this option removed?

Reply:
Because now we have the task it is kind of useless. Before the task we needed a way to trigger Xcribe to document or not when `mix test` was run. So a good way to do that was configure a ENV tha was present on CI to trigger Xcribe on CI environment. Now if you want to run Xcribe just need use the task.